interface/modules/custom_modules/oe-module-weno/scripts/weno_pharmacy_search.php

```diff
@@ -91,22 +92,28 @@
         $sql .= " AND test_pharmacy = ?";
         $params[] = $weno_test_pharmacies;
     }
+    if (!empty($weno_only)) {
+        $sql .= " AND on_weno = ?";
+        $params[] = $weno_only;
+    }
 
     $sql .= " ORDER BY Business_Name ASC";
 
     $return_arr = [];
     $res = sqlStatement($sql, $params);
     while ($row = sqlFetchArray($res)) {
-        $return_arr[] = array(
-            "name" => ucwords(strtolower($row['Business_Name'] . " " . $row['address_line_1'] . " " . $row['city'])),
-            "ncpdp" => $row['ncpdp']
-        );
+        if (strlen($row['ncpdp_safe']) < 8) {
+            $return_arr[] = array(
+                "name" => $row['Business_Name'] . " " . $row['address_line_1'] . " " . $row['city'] . ", " . $row['state'],
+                "ncpdp" => $row['ncpdp_safe']
+            );
+        }
     }
-    echo text(json_encode($return_arr));
+    echo json_encode($return_arr);
```

Comment:
Was text() call breaking something here? Hopefully not since we do this frequently (html escape a json_encoding value when echo'ing into html context)

Reply:
Yes there can be issues escaping with text here.  Main issue is this data comes from database and has special characters like '&' that can be part of a pharmacy name and thus you wind up with mangled(noticeable) escaped characters in select.
I let it slide but got called out about it twice but Weno testers.
I've had other times where wrapping with text can be an issue but not enought to be concerned and just have to be aware where used.